### PR TITLE
refactor!: remove learner_downloadable field

### PR DESCRIPTION
**Pull request by Ian2012**
This PR removes the `learner_downloadable` field.

Closes: #237

### CODE REVIEW

File: openedx_learning/apps/authoring/components/api.py [hunk 1]
```diff
@@ -422,7 +420,7 @@ def create_component_version_content(
     content_id: int,
     /,
     key: str,
-    learner_downloadable: bool = False,
+    learner_downloadable: bool = None,
```
Comment (ormsbee) [inline openedx_learning/apps/authoring/components/api.py:423]: Okay, change in plan: Let's just kill the argument entirely, and then bump the version to 0.7.0.

**Review by ormsbee — CHANGES_REQUESTED** [review]
Looks great! I have a minor request because a change in requirements on my part. Thank you!

File: openedx_learning/apps/authoring/components/api.py (hunk 1 above)
Comment (Ian2012) [reply openedx_learning/apps/authoring/components/api.py:423]: Sure

**Review by ormsbee — CHANGES_REQUESTED** [review]
Code looks good, but please bump the version to 0.17.0

**Pull request merged**